.canon.yaml

```
@@ -1,6 +1,6 @@
 rplidar:
-    image_amd64: ghcr.io/viamrobotics/canon:amd64
-    image_arm64: ghcr.io/viamrobotics/canon:arm64
+    image_amd64: ghcr.io/viamrobotics/rdk-devenv/canon:amd64
+    image_arm64: ghcr.io/viamrobotics/rdk-devenv/canon:arm64
```

```suggestion
    image_amd64: ghcr.io/viamrobotics/rdk-devenv:amd64
    image_arm64: ghcr.io/viamrobotics/rdk-devenv:arm64
```
Same as the file in RDK: https://github.com/viamrobotics/rdk/blob/main/.canon.yaml#L4-L5

---

ah I see my bad

---

should (hopefully) be all fixed now!